presto-router/src/main/resources/router_ui/src/webpack.config.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+    entry: {
+        'index': __dirname +'/index.jsx',
+    },
+    mode: "development",
+    module: {
+        rules: [
+            {
+                test: /\.(js|jsx)$/,
+                exclude: /node_modules/,
+                use: ['babel-loader']
+            }
+        ]
+    },
+    resolve: {
+        extensions: ['*', '.js', '.jsx']
+    },
+    output: {
+        path: __dirname + '/../dist',
+        filename: '[name].js'
```

Comment:
May we have a minimized JS (with map) in the future? That'll make the UI even more lightweight. It looks good to me for this version as the main Presto UI also does not have the minimized, productized version.

Reply:
Good call, that's really something we can have in the future.  Also, we might need share code between presto-server ui and presto-router ui.

Reply:
+1. The minimized JS bundles can significantly decrease the size of static JS files and accelerate the page loading. I also agree that some UI components can be reused in both presto-server UI and presto-router UI. That will make the code more robust. Let's put this in our plan.